soh/soh/SaveManager.cpp

```diff
@@ -543,7 +543,10 @@ void SaveManager::SaveGlobal() {
     globalBlock["zTargetSetting"] = gSaveContext.zTargetSetting;
     globalBlock["language"] = gSaveContext.language;
 
-    std::ofstream output("Save/global.sav");
+    const std::filesystem::path sSavePath(Ship::Window::GetPathRelativeToAppDirectory("Save"));
+    const std::filesystem::path sGlobalPath = sSavePath / std::string("global.sav");
```

Comment:
i copied these lines form `SaveManager::Init`, i feel like it's minimal copypasta so i'm not too concerned about it